src/main/java/org/owasp/esapi/configuration/EsapiPropertyManager.java

```diff
@@ -76,6 +76,18 @@ public Boolean getBooleanProp(String propertyName) throws ConfigurationException
         throw new ConfigurationException("Could not find property " + propertyName + " in configuration");
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Boolean getBooleanProp(String propertyName, Boolean defaultValue) {
+        for (AbstractPrioritizedPropertyLoader loader : loaders) {
+            return loader.getBooleanProp(propertyName, defaultValue);
```

Comment:
I think the way this is currently implemented, I don't think it will ever reach the final return on line 87 here. Maybe I'm wrong about that. Do we have a test for that case? Of course, if we change it as per my previous comment, we may still need it. Or may need to add a try / catch here and just rethrown any `ConfigurationException`.